source/declarators.tex

```diff
@@ -2257,7 +2257,7 @@
 \end{example}
 
 \begin{example}
-One can make a class uncopyable, i.e. move-only, by using deleted
+One can make a class uncopyable, i.e., move-only, by using deleted
```

Comment:
This seems like abuse of "i.e." -- uncopyable and move-only are not equivalent. More drastic rewording seems necessary here.